stl/inc/span

```diff
@@ -341,15 +341,15 @@ template <class _Rng>
 struct _Has_container_interface<_Rng,
     void_t<decltype(_STD data(_STD declval<_Rng&>())), decltype(_STD size(_STD declval<_Rng&>()))>> : true_type {};
 
-// STRUCT TEMPLATE _Is_span_compatible_range
+// VARIABLE TEMPLATE _Is_span_compatible_range_v
 // clang-format off
 template <class _Rng, class _Ty>
-struct _Is_span_compatible_range : bool_constant<conjunction_v<
+using _Is_span_compatible_range_v = conjunction_v<

```

Comment:
(1) s/using/inline constexpr bool/ (2) There's no reason to have `_v` in the name; `_Is_span_compatible_range` is a perfectly good name.

Reply:
Way better, thanks!